Source/WebCore/crypto/gcrypt/CryptoKeyRSAGCrypt.cpp

```diff
@@ -94,15 +94,9 @@ RefPtr<CryptoKeyRSA> CryptoKeyRSA::create(CryptoAlgorithmIdentifier identifier,
             return nullptr;
     }
 
-    CryptoKeyType keyType;
-    switch (keyData.type()) {
-    case CryptoKeyRSAComponents::Type::Public:
+    CryptoKeyType keyType = CryptoKeyType::Private;
+    if (keyData.isPublic())
```

Comment:
Ditto.  This change doesn't seem meaningful.

Reply:
removed.